### ISSUE TITLE
`/api/v1/query` returns wrong field names for scalar results

### ISSUE THREAD

**Issue by juliusv** [prometheus/prometheus#847]
Spec says:

```json
{
  "value": <sample_value>,
  "timestamp": <unix_time>
}
```

I'm getting:

```json
{
  "Value":"1",
  "Timestamp":1435229995.372
}
```

(fields shouldn't be capitalized)

**Comment by juliusv** [comment]
@fabxc

**Comment by fabxc** [comment]
Fixed in #848
